mindocr/models/necks/fpn.py

```diff
@@ -280,17 +280,17 @@ def __init__(self, in_channels, out_channels=128):
     def construct(self, features):
         f1, f2, f3, f4 = features
 
-        out = ops.ResizeBilinear(f3.shape[2:], True)(f4)
+        out = ops.ResizeBilinearV2(True)(f4, f3.shape[2:])
```

Comment:
Will operator initialization inside `construct` affect execution speed in `PyNative` mode? Will it be better to initialize the operator once in `__init__` (e.g. `self.resize = ops.ResizeBilinearV2(True)`) and then use it directly inside `construct`?

Reply:
Yes, it's better to define ops.ResizeBilinearV2(True) in __init__, then call it in the construct function. But I'm not sure if it will be a problem to export the existing ckpt file to MindIR after adding this line of code. The above codes are ok for the current ckpt file to export.

Reply:
Will try it now